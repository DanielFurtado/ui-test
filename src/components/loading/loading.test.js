import renderer from 'react-test-renderer';
import Loading from './loading.component';

it('renders', () => {
  const component = renderer.create(<Loading />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
