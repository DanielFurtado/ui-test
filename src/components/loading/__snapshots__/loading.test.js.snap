// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<h3
  className="loading"
>
  Loading
  <div
    className="lds-ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</h3>
`;
